interactions/models/internal/extension.py

```diff
@@ -128,6 +129,11 @@ def __new__(cls, bot: "Client", *args, **kwargs) -> "Extension":
             else:
                 raise TypeError("async_start is a reserved method and must be a coroutine")
 
+        instance.load(*args, **kwargs)
+        with contextlib.suppress(asyncio.CancelledError):
+            asyncio.get_running_loop()  # guard condition
+            _ = asyncio.create_task(instance.a_load(*args, **kwargs))
```

Comment:
`a_load` is a vague declaration name, I propose changing to `async_load` to clarify without having people guess or infer off of IntelliSense for async colouring

```suggestion
            _ = asyncio.create_task(instance.async_load(*args, **kwargs))
```

Reply:
Per conversation with Astrea. Approving this and related